tools/python/descriptions_downloader.py

```diff
@@ -143,15 +152,21 @@ def download_all(path, url, langs):
         download(path, lang[1])
 
 
-def worker(output_dir, langs):
+def worker(output_dir, checker, langs):
     @functools.wraps(worker)
     def wrapped(line):
+        if not line.strip():
+            return
+
         try:
-            url = line.rsplit("\t", maxsplit=1)[-1]
+            splitted = line.split("\t")
```

Comment:
```suggestion
            (mwm_path, ident, url) = line.split("\t")
```
Ну и дальше подправить. Так будет сильно читабельнее, чтобы не копаться с индексами (в этой функции у тебя используется одновременно и 1, -1

Reply:
++ так лучше